src/card.css

```
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
```

```suggestion
  width: 200px;
```
add box-sizing border-box on * selector